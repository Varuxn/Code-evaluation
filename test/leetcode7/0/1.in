9
593 850 633 326 649 934 455 679 656
