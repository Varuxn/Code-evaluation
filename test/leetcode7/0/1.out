267 217 307 326 194 479 455 23 656 